tests/test_cif_io.py

```diff
@@ -177,4 +177,21 @@ def mockreturn(*args, **kwargs):
         assert actual == expected
         # assert True
 
-# End of file.
+
+mpcontribs_tst = [
+    ('mpc', 'This function is not yet implemented'),
+    ('fs', None),
+    (False, 'mpc_client must be either mpc or fs')
+]
+
+
+@pytest.mark.parametrize("inpt, expected", mpcontribs_tst)
```

Comment:
I would change `inpt` to `input`.  Never miss an opportunity to make your code more readable by a human.

Reply:
Changing to input shadows the built-in keyword input so I left it as inpt but I will change it ltr!